lib/puppet/provider/iis_site/webadministration.rb

```diff
@@ -168,4 +168,11 @@ def self.to_bool(value)
     return :false  if value == false  || value =~ (/(^$|false|f|no|n|0)$/i)
     raise ArgumentError.new("invalid value for Boolean: \"#{value}\"")
   end
+
+  def binding_value
```

Comment:
A name that included `port` would be helpful I think, given you're fishing the port out of the binding string?